pip/src/interpreter.rs

```diff
@@ -113,7 +110,15 @@ impl Interpreter {
             TargetProfile::Unrestricted => Profile::Unrestricted,
             TargetProfile::Base => Profile::Base,
         };
-        let language_features = language_features.unwrap_or_default();
+        // If no features were passed in as an argument, use the features from the manifest.
+        // this way we prefer the features from the argument over those from the manifest.
+        let language_features: Vec<String> = match (language_features, &manifest_descriptor) {
+            (Some(language_features), _) if !language_features.is_empty() => language_features,
```

Comment:
I was just replicating the already existing behavior in the Python file, which is loading the settings from the manifest if `language_features` is an empty list.

![image](https://github.com/microsoft/qsharp/assets/156957451/2705896e-916b-42ea-98ae-1e46aee6ffb1)

Reply:
However, what you suggest makes more sense than the previous behavior. I'm guessing we want to change the default argument of the init function to `None` instead of the current `[]`.

Reply:
Fixed this.